### DISCUSSION TITLE
Making Stream Deck even better

### DISCUSSION THREAD

**Discussion by UweSchwo** [elgatosf/elgato#8] [Ideas]
I'm not sure if this is the right place to make suggestions concerning my Stream Deck XL core software. 
If the place here concerns only Add On Developers - where should I search instead?

So I'll try it out:
It would be great, if the Stream Deck Console (StreamDeck.exe)  would have the possibility to start with command line parameters concerning the device (which Deck) and the profile I'd like to start with.
Greetings, Uwe

**Comment by ChekTek** [comment]
Great suggestion! I'm afraid this is a real power user feature, which can be challenging to get prioritized, but I will take this feature request to the Stream Deck core team.
